Artifacts/windows-install-certificate/Artifactfile.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+  "title": "Install Certificate",
+  "publisher": "Ross Bohr",
+  "description": "Installs a certificate in the LocalMachine/My store.",
+  "tags": [
+    "Certificate",
+    "PFX",
+    "CurrentUser",
+    "Windows"
+  ],
+  "iconUri": "https://www.iconninja.com/files/305/837/188/certificate-icon.png",
+  "targetOsType": "Windows",
+  "parameters": {
+    "certificateName": {
+      "type": "string",
+      "displayName": "Certificate Name",
+      "description": "The name of the certificate which must be installed. This property is used to distinguish multiple instances of this artifact.",
+      "defaultValue": "NewCertificate"
+    },
+    "base64cert": {
+      "type": "string",
```

Comment:
You could consider changing the type to `fileContents` and it would allow you to upload a file instead. Just another possibility. Take a look at the artifact [`Test Parameter Types`](https://github.com/Azure/azure-devtestlab/tree/master/Artifacts/windows-test-paramtypes).

Reply:
It is convenient to store the pfx file in base64 string together with the artifact instance in a DevTest Lab formula instance. This is the same approach MS follows for Service Fabric cluster connectors in VSTS, thus a standard one. I'd not change it.

Reply:
![azure_devtestlab_formula](https://user-images.githubusercontent.com/8200018/35974988-d841f9d0-0ce3-11e8-9c53-796dcb82c738.png)
A sample formula in Azure DevTest Lab as seen in Azure Resource Explorer.

Reply:
It would be nice to use "fileContents", however it seems it has the following limitation:
"Validation error for mycert.pfx. Details: Selected file 'mycert.pfx' must be between 1 and 2048 bytes."

Reply:
@leovms  , is there a possibility to override the default size limit (1-2048 bytes) for "fileContents" ? If not, then it is not practical to use this type.

Reply:
No, it seems we're limited to that size.